examples/AlwaysMints.purs

```diff
@@ -29,6 +27,10 @@ import Contract.Value as Value
 import Contract.Wallet (mkNamiWalletAff)
 import Data.BigInt as BigInt
 
+import Effect.Aff (launchAff_)
```

Comment:
Please sort imports and remove empty lines.

Reply:
Still needs to be done